wordpress-mobile/WordPress-Android
# Reformat jetpack package files

This PR reformats all the classes (+ corresponding test classes) in the `org.wordpress.android.ui.jetpack` package.

To test:
- Right-click on the `jetpack` folder (`org.wordpress.android.ui.jetpack` package).
- Click Reformat Code from the options.
- Repeat the above for the test classes under the same package.
- Notice that no files should be changed.

## Regression Notes
1. Potential unintended areas of impact
N/A - an IDE formatter on the package

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A 

3. What automated tests I added (or what prevented me from doing so)
N/A 

PR submission checklist:
